### PR TITLE
fix: delete demo-registry in registry plugin and fix broken link

### LINKED ISSUE #1000: 修复失效链接和删除插件中多余demo应用
### What happened?

1. 发现一些失效链接
2. 注册插件存在多余demo应用

### How can we reproduce it (as minimally and precisely as possible)?

不相关

### Anything else we need to know?

_No response_

### Sermant version

不相关

### OS version

不相关

### PR DESCRIPTION
**Pull request by robotLJW**
What type of PR is this?

/fix

Which issue(s) this PR fixes:

Fixes #1000 

1. delete demo-registry in registry plugin
2. demo module's link is dead
3. fix broken link in registration plugin docs

Use case description: not involving

Does this PR introduce a user-facing change?

No

Additional documentation e.g., usage docs, etc.:

No

### CODE REVIEW

**Review by lilai23 — DISMISSED** [review]

File: docs/user-guide/registry/document-zh.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ nacos.service:
 
 注意：
 1、nacos的group通过核心配置文件的service.meta.application设置
-2、nacos参数目前仅展示常用参数，其他参数项见[NACOS配置文件](../../../sermant-plugins/sermant-service-registry/registry-common/src/main/java/NacosRegisterConfig.java)
+2、nacos参数目前仅展示常用参数，其他参数项见[NACOS配置文件](https://github.com/huaweicloud/Sermant/blob/develop/sermant-plugins/sermant-service-registry/registry-common/src/main/java/com/huawei/registry/config/NacosRegisterConfig.java)
```
Comment (provenceee) [inline docs/user-guide/registry/document-zh.md]: 这里要改成相对路径

File: docs/user-guide/registry/document.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ nacos.service:
 
 Notice: 
 1. group of nacos can setting by core config service.meta.application.
-2. nacos configs current only show normal use type, others see [NACOS config file](../../../sermant-plugins/sermant-service-registry/registry-common/src/main/java/NacosRegisterConfig.java)
+2. nacos configs current only show normal use type, others see [NACOS config file](https://github.com/huaweicloud/Sermant/blob/develop/sermant-plugins/sermant-service-registry/registry-common/src/main/java/com/huawei/registry/config/NacosRegisterConfig.java)
```
Comment (provenceee) [inline docs/user-guide/registry/document.md]: 这里改成相对路径

File: docs/user-guide/registry/document-zh.md (hunk 1 above)
Comment (robotLJW) [reply docs/user-guide/registry/document-zh.md]: 已经修改了

**Review by provenceee — APPROVED** [review]

**Review by JoeMinty — APPROVED** [review]
LGTM

**Pull request merged**